docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -1510,6 +1510,33 @@ Parse data with JSONparse.
 {{/JSONparse}}
 <!-- output: bar -->
 ```
+### {{JSONparseSafe}}
+
+Parse data safely with JSONparseSafe. This helper is similar to the JSONparse helper, but allows handling of incorrect JSON data. Returns `parsed data` when JSON is correct; otherwise, renders an `else` block.
+
+
+#### Parameters
+
+- `json` {String} 
+
+
+#### Example
+
+```handlebars
+{{#JSONparseSafe '{"foo": "bar"}'}
+  {{foo}}
+{{/JSONparseSafe}}
+<!-- output: bar -->
+```
+
+```handlebars
+{{#JSONparseSafe 'foo'}
+  {{foo}}
+{{else}}
+  John Doe
+{{/JSONparseSafe}}
+<!-- output: bar -->
+```
 

```

Comment:
add a link to the helper in github -- for accessible link format, see strReplace GitHub link.

Reply:
@slsriehl I could not find `JSONParse` or `JSONParseSafe` in the list of [helpers](https://github.com/bigcommerce/paper-handlebars/tree/master/helpers).

Reply:
I could not find JSONParseSafe in GitHub. It is not on the list of [helpers](https://github.com/bigcommerce/paper-handlebars/tree/master/helpers).

Reply:
It can be available, once [PR](https://github.com/bigcommerce/paper-handlebars/pull/235) will be merged

Reply:
Thank you @RomanKrasinskyi I will add a link to it.